src/main/java/com/feywild/feywild/block/ModBlocks.java

```diff
@@ -38,4 +38,6 @@
     public static final AncientRunestoneBlock ancientRunestone = new AncientRunestoneBlock(FeywildMod.getInstance());
 
     public static final DisplayGlassBlock displayGlass = new DisplayGlassBlock(FeywildMod.getInstance());
+
+    //   public static final RotatedPillarBlock strippedAutumnTreeLog = new RotatedPillarBlock(BlockBehaviour.Properties.copy(Blocks.STRIPPED_OAK_LOG));
```

Comment:
Added but commented out code. Either add a comment what it is and why it was added or remove it.